### PR TITLE
delete rule SLES-15-040280

### PR DESCRIPTION
**Pull request by svet-se**
#### Description:

- _Delete rule sshd_disable_compression_

#### Rationale:

- _Accoring DISA recommendations Version 1, Release 9 from 26 January 2023 about SLE 15 STIG - "SLES-15-040280 - Deleted rule as it is not applicable to SLES 15."_

### CODE REVIEW

File: linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml [hunk 1]
```diff
@@ -23,7 +23,6 @@ identifiers:
     cce@rhel8: CCE-80895-6
     cce@rhel9: CCE-90801-2
     cce@sle12: CCE-83062-0
-    cce@sle15: CCE-85647-6
```
Comment (marcusburghardt) [inline linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml:26]: It is not a good idea to remove a `CCE` from a rule. In general, the once a rule receives a `cce`, they are tied forever.
https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#rules

**Review by marcusburghardt — CHANGES_REQUESTED** [review]
There is only one point about the `cce` removal.

File: linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml (hunk 1 above)
Comment (svet-se) [reply linux_os/guide/services/ssh/ssh_server/sshd_disable_compression/rule.yml]: Thanks, @marcusburghardt ,

I've reverted the CCE changes.

Best regards and have a nice day!

**Review by marcusburghardt — APPROVED** [review]
Thanks

**Comment by marcusburghardt** [comment]
> @marcusburghardt FYI I'm working for SUSE (with @rumch-se) and will contribute to this project. Please advise how can I be added to ComplianceAsCode/suse-maintainers.

Welcome @svet-se . First, if you are not already there, I would like to invite you to join us in [Gitter](https://matrix.to/#/#Compliance-As-Code-The_content:gitter.im). There you will find most of the contributors and the maintainers.

Regarding the criteria to get merge rights, please take a look in our Governance policies:
https://complianceascode.readthedocs.io/en/latest/manual/developer/01_introduction.html#governance

You can approach us if you have any question.

**Comment by marcusburghardt** [comment]
Overriding CODEOWNERS since a SUSE approver is not currently available.

**Pull request merged**